packages/components/src/interfaces/complex/DynamicStatistics.ts

```diff
@@ -0,0 +1,80 @@
+import type { Static } from "@sinclair/typebox"
+import { Type } from "@sinclair/typebox"
+
+export const DYNAMIC_STATISTICS_TYPE = "dynamicstatistics"
```

Comment:
Note: im proposing for us to define the type as a const.

Right now, the existing components hardcode it as a string literal which can be less intuitive, especially when they are used in keys.

Reply:
then should these const definitions be in a separate file where it defines all the blocks?